sfge/src/main/java/com/salesforce/apex/jorje/MethodWrapper.java

```diff
@@ -36,6 +35,10 @@ protected void fillProperties(Map<String, Object> properties) {
             // Only add if it is present
             properties.put(Schema.IS_TEST, true);
         }
+        if (isImplicitDefaultConstructor()) {
+            // Only add if necessary.
+            properties.put(Schema.IS_IMPLICIT, true);
```

Comment:
As per general convention, boolean property is only added on vertices where it's `true`, to save space.